In [4]:
### Q3.52
install.packages('Stat2Data', repos='http://cran.us.r-project.org')
library(Stat2Data)
data(LewyBody2Groups)
str(LewyBody2Groups)

Installing package into ‘/home/jupyter/R/x86_64-pc-linux-gnu-library/4.2’
(as ‘lib’ is unspecified)



'data.frame':	39 obs. of  3 variables:
 $ Type: Factor w/ 2 levels "DLB","DLB/AD": 1 1 1 1 1 1 1 1 1 1 ...
 $ APC : num  0.85 0.49 0.12 0 -0.22 -0.35 -0.4 -0.41 -0.43 -0.45 ...
 $ MMSE: num  2.22 0.37 -0.1 -2.99 0.66 -2.47 -0.03 -1.92 -0.09 -3.03 ...


In [5]:
# part a
in_model = lm(MMSE~APC+Type+APC*Type, data=LewyBody2Groups)
summary(in_model)


Call:
lm(formula = MMSE ~ APC + Type + APC * Type, data = LewyBody2Groups)

Residuals:
    Min      1Q  Median      3Q     Max 
-8.3905 -1.5841 -0.1014  1.6959  4.9309 

Coefficients:
               Estimate Std. Error t value Pr(>|t|)  
(Intercept)     -0.5846     0.7927  -0.738   0.4657  
APC              2.3176     1.1640   1.991   0.0543 .
TypeDLB/AD      -1.8513     1.1471  -1.614   0.1155  
APC:TypeDLB/AD  -0.9732     1.2712  -0.766   0.4490  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 2.64 on 35 degrees of freedom
Multiple R-squared:  0.3484,	Adjusted R-squared:  0.2926 
F-statistic: 6.239 on 3 and 35 DF,  p-value: 0.001656


For type DLB/AD: MMSE-hat = -0.5846 + 2.3176APC - 1.8513*1 - 0.9732APC*1 = -2.4359 + 1.3444APC \
For type DLB: MMSE-hat = -0.5846 + 2.3176APC

part b \
t = -0.766, df = 35, p-value = 0.4490, fail to reject H0. \
We believe that INTERACTION is not needed and that parallel lines should suffice.

In [6]:
# part c
reduced_model = lm(MMSE~APC,data=LewyBody2Groups)
anova(reduced_model, in_model)

Res.Df RSS      Df Sum of Sq F        Pr(>F)   
1 37     262.5764 NA       NA        NA        NA
2 35     243.8750  2 18.70134  1.341972 0.2744446

F = 1.342, P-value = 0.2744, fail to reject H0, we have no evidence that the APC can affect MMSE.
The term involving Type can affect MMSE